tools/hansel/bio_hansel.xml

```diff
@@ -1,33 +1,28 @@
-<tool id="bio_hansel" name="Hansel - Salmonella Subtyping" version="0.1.0">
```

Comment:
Need to have a short description line :)

Reply:
Description added!